src/containers/Articles/Article/ReportArticleContainer.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { reportArticle } from '../../../store/actions/async/ReportArticle';
+import ReportArticle from '../../../components/UI/Article/ReportArticle';
+
+export class ReportArticleContainer extends Component {
+    state = {
+        reason: ''
+    };
+
+    onChangeHandler = (e, { value }) => this.setState({ reason: value });
+
+    onSubmitHandler = e => {
+        e.preventDefault();
+        const { 
+            reportArticle, 
+            match: { params: { slug } = {} } = {}, 
```

Comment:
@oma0256 this destructuring is hard to follow.

Reply:
@collinewait had advised me to use the same on my previous PR,` const { match: {params: {slug}} = {params: {}} } = this.props;` I think it is okay.